Documentation/api-reference.md

```diff
@@ -462,6 +588,23 @@ AuthenticatedState state = item.getAuthenticatedState();
 boolean primary = item.isPrimary();
 ```
 
+#### Kotlin
+
+```kotlin
+// Construct
+val item = IdentityItem("identifier")
+
+val item = IdentityItem("identifier", AuthenticatedState.AUTHENTICATED, false)
+
+
+// Getters
+val id = item.getId()
+
+val state = item.getAuthenticatedState()
+
+val primary = item.isPrimary()
```

Comment:
we can use property access syntax for these getters 
val id = item.id
val state = item.authenticatedState
val primary = item.isPrimary